### PR TITLE
Add new "The Legend of Capturing" map by gameit

### PR DESCRIPTION
**Pull request by JostP**

### CODE REVIEW

**Comment by TSafa-23** [comment]
Looks great. Please merge soon!

**Review by TSafa-23 — APPROVED** [review]

**Comment by JostP** [comment]
Needs better screenshot.

**Comment by Avyukt1** [comment]
What's this?
image

**Comment by Avyukt1** [comment]
@JostP 
The [plane_crash](https://github.com/MT-CTF/maps/pull/124) map has made some changes in this map's outer side. Maybe you should take a look.

**Comment by Avyukt1** [comment]
image
image
image
 One base is inside a castle and has more protection than the other one which is on a bridge.
image
image

File: capture_legend/map.conf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name = The Legend of Capturing
+author = gameit
+
+r = 87
```
Comment (Avyukt1) [inline capture_legend/map.conf:4]:
```
r = 87
h = 108
```
These lines are creating error!

File: capture_legend/map.conf [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+h = 108
```
Comment (Avyukt1) [inline capture_legend/map.conf:5]: These lines are creating errors
```
r = 87
h = 108
```

Comment (JostP) [reply capture_legend/map.conf:5]: I don't think so. I get no errors. You will get error if h isn't even number.

Comment (Avyukt1) [reply capture_legend/map.conf:5]: See the screenshot
https://github.com/MT-CTF/maps/pull/185#issuecomment-752921597

Comment (Avyukt1) [reply capture_legend/map.conf:5]: I don't even have the plane_crash in that folder anymore.

Comment (Avyukt1) [reply capture_legend/map.conf:5]: image
This is outside the barrier.

Comment (Avyukt1) [reply capture_legend/map.conf:5]: image

**Comment by JostP** [comment]
I have to say that this map is really good :+1:.
But it still have some issues:
- door c. It would be great if it was removed with next update, but it is ok for now.
Zaslonska slika 2020-12-31 15-56-00
- Blue base is a lot more protected than red's. Like Avyukt1 said. I am sure that this will have to be updated in the near future as it is quite serious issue. But I think we can still add it and see what players say.

Zaslonska slika 2020-12-31 15-48-48

Zaslonska slika 2020-12-31 15-48-53

**Comment by JostP** [comment]
I think custom chest zones would be a great addition.

File: capture_legend/map.conf (hunk 2 above)
Comment (JostP) [reply capture_legend/map.conf:5]: create new world and then try.

Comment (Avyukt1) [reply capture_legend/map.conf:5]: Yes, new world is working fine.

Comment (JostP) [reply capture_legend/map.conf:5]: this happened because you created new world with plane crash included.

**Review by Avyukt1 — COMMENTED** [review]
Yes, This map will be 1 of the best ones in CTF.

**Pull request merged**